recipe/meta.yaml

```diff
@@ -20,12 +19,12 @@ requirements:
     - pip
     - pytest-runner
     - setuptools >=20.0
-    - wheel
+    - wheel !=0.37.1
   run:
     - python
-    - setuptools >=20.0
+    - setuptools !=62.6
     - lazy-object-proxy >=1.4.0
-    - wrapt >=1.11,<1.14
+    - wrapt >=1.11,<2
     - typed-ast >=1.4.0,<2.0  # [py<38]
     - typing-extensions >=3.10 # [py<310]
 
```

Comment:
I guess we should add `run_constrained`:
  `- python >=3.7.2`
  see https://github.com/PyCQA/astroid/blob/14a0d657809ab5b42e7d416b686953508e6d8b91/pyproject.toml#L33
That can be discussed

Reply:
It would be nice to update test/imports and check if it'll work:
```
  imports:
    - astroid
    - astroid.brain
    - astroid.interpreter
    - astroid.nodes 
```

see https://github.com/PyCQA/astroid/tree/v2.12.2/astroid and https://github.com/PyCQA/astroid/blob/14a0d657809ab5b42e7d416b686953508e6d8b91/pyproject.toml#L52

Reply:
**Home url** https://www.astroid.org/ is outdated. It should be https://github.com/PyCQA/astroid